nautilus_trader/persistence/catalog/parquet.py

```diff
@@ -541,7 +541,7 @@ def _build_query(
 
         if start:
             start_ts = dt_to_unix_nanos(start)
-            conditions.append(f"ts_init >= {start_ts}")
+            conditions.append(f"ts_init > {start_ts}")
```

Comment:
Surely we want to be inclusive of the start?

Reply:
Having end inclusive I don't think start should be included.

Reply:
Whats your reasoning here? I think this would be surprising behaviour to be start exclusive - normally the choice is `end` inclusive or exclusive (but `start` is always inclusive for most APIs).

Reply:
One reason we wouldn't want to make this change is Databento is `start` inclusive and `end` exclusive.

Reply:
I'm open to adopting the standard of making the end exclusive as well. My main concern arises when both start and end points are inclusive, as that's where potential issues of duplicate record.

Reply:
In that case, we just need one end exclusive - so I think better for that to be `end` for consistency.

It can still be surprising to some users though to have exclusive endpoints in ranges.

Reply:
Understood, it seems this issue wouldn't typically arise during normal use. Additionally, I've identified a similar situation elsewhere in the catalog. To sidestep the complexity of adding flags and the potential impact on other users, I'll close this matter for now.

Reply:
You make a good point about duplicate records _if_ the catalog is repeatedly queried, as start and end are both inclusive. I'll have a deeper think on this and look at the use cases.